src/utils/composeStores.js

```diff
@@ -1,11 +1,18 @@
-import mapValues from '../utils/mapValues';
-import pick from '../utils/pick';
+import mapValues from './mapValues';
+import pick from './pick';
+
+// UNDEF = undefined;
+const UNDEF = void 0;
```

Comment:
Unfortunately, there is an ESLint rule that forbids `return;` when there is another type returned elsewhere. I think we can just use `undefined` though. We don't have to worry about users overwriting `undefined` in tests.